src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts

```diff
@@ -239,6 +239,7 @@ export class ActivitySearchComponent implements OnInit, OnDestroy {
     /* istanbul ignore else */
     if (_.get(this.queryParams, 'key')) {
       option.query = this.queryParams.key;
+      option.filters = _.pick(filters, 'primaryCategory', 'objectType', 'status');
```

Comment:
Why this change is added now. WHat is the root cause for this?
cc: @balakrishna10